File: Java/LT01_ESTSEQ/LT01_ESTSEQ_24.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */

/**
 * Nome do Programador: Lucas Sartorelli
 * Data: 22/02/2025
 * Descrição:
 * Receba um valor inteiro. 
 * Verifique e mostre se é divisível por 2 e 3.
 */
import javax.swing.JOptionPane;
public class LT01_ESTSEQ_24 {
    public static void main(String args[]){
        Integer num;
        num=Integer.parseInt(JOptionPane.showInputDialog("Insira o Número a ser Testado"));
        if(((num%2)==0)&&((num%3)==0)){
            JOptionPane.showMessageDialog(null, num +" é divisivel por 2 e 3.");
        }else{
            JOptionPane.showMessageDialog(null, num +" não é divisivel por 2 e 3.");
        }
    }
}

File: C/lt_01_estseq/lt01_estseq_24.c
#include <stdio.h>

int main() {

  int num;

  printf("Insira o numero a ser testado: ");
  scanf("%d", &num);

  if (((num % 2) == 0) && ((num % 3) == 0)) {
    printf("%d e divisivel por 2 e 3.\n", num);
  } else {
    printf("%d nao e divisivel por 2 e 3.\n", num);
  }

  return 0;
}
